modules/nf-core/ctree/tests/main.nf.test

```diff
@@ -0,0 +1,93 @@
+nextflow_process {
+
+    name "Test Process CTREE"
+    script "../main.nf"
+    process "CTREE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ctree"
+
+    test("test mobster rds input") {
+        when {
+            process {
+                """
+                meta = [ id : "test" ]
+                ctree_input = file(params.modules_testdata_base_path + "delete_me/ctree/HCC1395_WXS_HCC1395_HCC1395_HCC1395T_mobster_st_best_fit.rds", checkIfExists: true)
+                input[0] = tuple(meta, ctree_input)
```

Comment:
Can you make this just 
`input[0] = [[ id : "test" ], file(params.modules_testdata_base_path + "delete_me/ctree/HCC1395_WXS_HCC1395_HCC1395_HCC1395T_mobster_st_best_fit.rds", checkIfExists: true)]` but indented nicer (see basically every other module)